examples/simple/windows/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/projectdiscovery/gozero/sandbox"
+)
+
+func main() {
+	commands := []string{
+		"ipconfig",
+	}
+	cfg := sandbox.Configuration{
+		Networking: sandbox.Enable,
+		VirtualGPU: sandbox.Default,
+		MemoryInMB: 500,
+	}
+	cfg.LogonCommands.Command = commands
```

Comment:
- same here Sanbox Configuration fields seems to have changed

Reply:
It's still platform specific. Probably later when integrating sandbox, we can define better a common abstract representation